ggircs-app/app/components/LoginButton.tsx

```diff
@@ -12,21 +11,25 @@ const LoginButton: React.FunctionComponent = () => {
     )}`;
 
   return (
-    <Form inline action={loginURI} method="post">
-      <button type="submit">Log in</button>
+    <form action={loginURI} method="post">
+      <input type="submit" value="Log in" />
```

Comment:
why is this an input now and not a button?

Reply:
I figured it would be easier to style, but it's actually the same